### PR TITLE
Pavi

### PR DESCRIPTION
**Pull request by pavikunusilin**
Not the completed version,but to avoid future conflicts.

### CODE REVIEW

**Comment by aaronmyatt** [comment]
@pavi304 tested this our and everything is working 👍

File: db/migrate/20181218081452_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+
```
Comment (aaronmyatt) [inline db/migrate/20181218081452_create_users.rb:4]: Do make a migration to add the email column to the user model

**Review by aaronmyatt — CHANGES_REQUESTED** [review]

File: db/migrate/20181218081452_create_users.rb (hunk 1 above)
Comment (pavikunusilin) [reply db/migrate/20181218081452_create_users.rb:4]: DONE. :)

File: app/controllers/home_controller.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@ def index
 
   def form
     session[:email]=params[:email]
+
+    # byebug
```
Comment (aaronmyatt) [inline app/controllers/home_controller.rb]: Let's remove this comment please

**Review by aaronmyatt — CHANGES_REQUESTED** [review]

File: app/controllers/home_controller.rb (hunk 2 above)
Comment (pavikunusilin) [reply app/controllers/home_controller.rb]: DONE. :)

**Comment by aaronmyatt** [comment]
@pavi304 looks good, shall go ahead and merge, I'll help Prosper with his branch if there are conflicts

**Pull request merged**